client/verta/verta/registry/entities/_modelversion.py

```diff
@@ -118,7 +118,7 @@ def __repr__(self):
                 "registered model id: {}".format(msg.registered_model_id),
                 "experiment run id: {}".format(msg.experiment_run_id),
                 # "archived status: {}".format(msg.archived == _CommonCommonService.TernaryEnum.TRUE),
-                "artifact keys: {}".format(artifact_keys),
```

Comment:
This would be redundant with the prettified artifact string which includes this information in addition to the upload status.